src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfoOfT.WriteHelpers.cs

```diff
@@ -301,10 +301,7 @@ public partial class JsonTypeInfo<T>
                 using var bufferWriter = new PooledByteBufferWriter(Options.DefaultBufferSize);
                 using var writer = new Utf8JsonWriter(bufferWriter, Options.GetWriterOptions());
 
-                if (!bufferWriter.CanGetUnflushedBytes)
-                {
-                    ThrowHelper.ThrowInvalidOperationException_PipeWriterDoesNotImplementUnflushedBytes(bufferWriter);
```

Comment:
Is the specific helper being used elsewhere still?

Reply:
Yes, there are two call sites, this is removing one of them.